Directory.Build.targets

```diff
@@ -1,7 +1,6 @@
 <Project>
   <PropertyGroup>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">$(MicrosoftNETCoreApp20PackageVersion)</RuntimeFrameworkVersion>
```

Comment:
Is 2.0 used anywhere?

Reply:
In [benchmarks](https://github.com/aspnet/Common/blob/dev/benchmarks/Microsoft.Extensions.ObjectPool.Performance/Microsoft.Extensions.ObjectPool.Performance.csproj).

Reply:
I forget. Was there a reason this was skipped?

Reply:
I was given to believe that we wanted to keep the benchmark stuff unchanged to preserve historical data. If this is not so we should add it to [this issue](https://github.com/aspnet/Universe/issues/1153).

Reply:
You can update this project. I double checked with @mikeharder and @pakrym. I think EF's benchmarks and https://github.com/aspnet/Benchmarks are the exceptions.